packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -397,7 +377,28 @@ describeCompat("Message size", "NoCompat", (getTestObjectProvider, apis) => {
 						runtimeOptions: {
 							...containerConfig.runtimeOptions,
 							maxBatchSizeInBytes: 51 * bytesPerKB, // 51 KB
-							chunkSizeInBytes: Infinity,
+							chunkSizeInBytes: Number.POSITIVE_INFINITY,
+						},
+					});
+
+					const largeString = generateRandomStringOfSize(maxMessageSizeInBytes);
+					const messageCount = 3; // Will result in a 150 KB payload
+					setMapKeys(localMap, messageCount, largeString);
+					await provider.ensureSynchronized();
```

Comment:
It always feels a little weird that there's no assert here, but I guess it's fine

Reply:
The "assert" in this case is the "itExpects" error on container close

Reply:
Feel free to resolve